pkg/events/derive/container_create.go

```diff
@@ -15,6 +16,10 @@ func ContainerCreate(containers *containers.Containers) deriveFunction {
 
 func deriveContainerCreateArgs(containers *containers.Containers) func(event trace.Event) ([]interface{}, error) {
 	return func(event trace.Event) ([]interface{}, error) {
+		// if cgroup_id is from non default hid (v1 case), this isn't a container, so we can skip
```

Comment:
actually, this is a container, but we want to avoid duplicates and searches when not the default hid